packages/MachineLearning/src/ValueObjects/AiCapabilityStatus.php

```
@@ -0,0 +1,109 @@
+            throw AiRuntimeContractException::invalidValue('AI capability status criticality');
+        }
+
+        return new self(
+            status: $status,
+            available: $data['available'],
+            fallbackUiMode: $fallbackUiMode,
+            messageKey: $messageKey,
+            reasonCodes: $reasonCodes,
+            operatorCritical: $data['operator_critical'],
+        );
+    }
```

PHPMD (2.15.0)

[warning] The method fromArray() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)